### PR TITLE
fix(FEC-7873): player shows audio and video flavour and in case of choosing audio flavour player stuck

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Send the hls `this` levels instead of the event data levels (the `this._hls.levels` are the filtered levels from the ManifestParsed event (MANIFEST_PARSED sent before MANIFEST_LOADED).

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment 
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [x] Docs have been updated

### CODE REVIEW

File: src/hls-adapter.js [hunk 1]
```diff
@@ -209,9 +209,9 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
    * @private
    */
   _parseTracks(data: any): Array<Track> {
-    let audioTracks = this._parseAudioTracks(data.audioTracks || []);
-    let videoTracks = this._parseVideoTracks(data.levels || []);
-    let textTracks = this._parseTextTracks(this._hls.subtitleTracks || []);
+    const audioTracks = this._parseAudioTracks(data.audioTracks || []);
```
Comment (OrenMe) [inline src/hls-adapter.js]: why not take audioTracks from [hls.audioTracks](https://github.com/video-dev/hls.js/blob/master/doc/API.md#hlsaudiotracks) as well? a bit strange we take other data but not this

Comment (dan-ziv) [reply src/hls-adapter.js]: Good point.
Done

Comment (OrenMe) [reply src/hls-adapter.js]: 👍 did you check all data is actually available at that point? should be but better ask now...

Comment (dan-ziv) [reply src/hls-adapter.js]: Better to be safe then sorry :)
Sure I've checked

**Comment by dan-ziv** [comment]
@OrenMe  fixed

**Pull request merged**